src/superphot_plus/lightcurve.py

```diff
@@ -0,0 +1,188 @@
+"""A data class for storing information for a single light curve."""
+import numpy as np
+
+from superphot_plus.file_utils import read_single_lightcurve, save_single_lightcurve
+
+
+class Lightcurve:
+    def __init__(self, times, fluxes, flux_errors, bands, name=None):
+        """A class for storing and manipulating a light curve.
+
+        Parameters
+        ----------
+        times : numpy array
+            The time stamps of the light curve data
+        fluxes : numpy array
+            The light curve fluxes
+        flux_errors : numpy array
+            The light curve flux errors
+        bands : numpy array
+            The band labels
+        name : str, optional
+            The name of the light curve.
```

Comment:
should we also include supernova type here? Maybe incorporate it with Sandro's recent SupernovaType() object type merge.

Reply:
Good call. That will allow us to store the label data in the same object if it is known. 

For now I added an optional input argument and an attribute to store it using an integer as a place holder. Once Sandro's PR goes in, I can upgrade this to a SupernovaType object if we want.

Reply:
This will make it easier to propagate the Lightcurve object into classification and physical parameter mapping